README.md

```diff
@@ -8,6 +8,20 @@ For more information on building and developing the core of Scala, read on! Plea
 We're still using Jira for issue reporting, so please [report any issues](https://issues.scala-lang.org) over there.
 (We would love to start using GitHub Issues, but we're too resource-constrained to take on this migration right now.)
 
+# Get in touch!
+If you need some help with your PR at any time, please feel free to @-mention anyone from the list below, and we will do our best to help you out:
+
+ username     |      full name       | talk to me about...                               |
+--------------|----------------------|---------------------------------------------------| 
+ adriaanm     | Adriaan Moors        | anything (type checker, pattern matcher, CI)      |
+ gkossakowski | Grzegorz Kossakowski | infrastructure, incremental compilation, back-end |
+ retronym     | Jason Zaugg          | Java 8 lambdas, tricky bug detective work         |
+ Ichoran      | Rex Kerr             | the collections library                           |
+ lrytz        | Lukas Rytz           | optimizer, named & default arguments              |
```

Comment:
I've tinkered with the REPL and also rewritten parts; but really it ought to be spun off.

You could imagine it gaining community momentum from sbt: someone just posted a (putative) REPL issue on the sbt repo.

Anyone taking REPL seriously wants to rewrite it, in whole or in part.

BTW, REPL stands for "Rewrite Every Paulp Lambda". And PAULP stands for "Please Allow Us Less Programming."